build.sbt

```diff
@@ -15,11 +15,22 @@ ThisBuild / tlSonatypeUseLegacyHost := false
 // publish website from this branch
 ThisBuild / tlSitePublishBranch := Some("main")
 
+ThisBuild / scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.6.0"
+
+ThisBuild / semanticdbEnabled := true
+ThisBuild / semanticdbVersion := scalafixSemanticdb.revision
+
+ThisBuild / tlCiScalafixCheck := true
```

Comment:
At this point you can remove these settings and just use `sbt-typelevel-scalafix` instead :)

Reply:
Indeed. Looks much better now, thanks 🙂